requirements.txt

```diff
@@ -2,3 +2,4 @@ PlexAPI>=4.2.0
 fuzzywuzzy
 pypiwin32
 numpy
+configargparse
```

Comment:
you should add `ConfigArgParse` to the requirements file.
```suggestion
ConfigArgParse
```
https://pypi.org/project/ConfigArgParse/ this is the package you are using right?

Reply:
yes, that's the one